api/payara-bom/pom.xml

```diff
@@ -483,7 +483,7 @@
             <dependency>
                 <groupId>org.eclipse.angus</groupId>
                 <artifactId>angus-activation</artifactId>
-                <version>${angus.activation.version}</version>
+                <version>${angus-activation.version}</version>
```

Comment:
By my suggested "use project names" convention this should probably just be `angus.version`

Reply:
You're correct!
We should probably leave this at it is then